Documentation/Group Meeting Minutes

```diff
@@ -0,0 +1,6 @@
+Group Meeting Minutes
+
+October 2nd
+
+12:30pm-1:10pm:Created Feature Hierarchy 
+1:10-2:30: created new features
```

Comment:
Couple notes here:
1. This file doesn't have a file type. I would recommend .md which is a Markdown file. You can see our other .md files for examples.
2. This could use some formatting standardization; things like word capitalization, including am/pm in times, spacing
3. Each time entry should probably have more details. What are the general concepts we talked about, what issues did we review, etc

Reply:
I will change it. Thank you:)